account_asset_disposal/models/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from . import account_asset_category
 from . import account_asset_asset
 from . import account_move_line
+from . import account_asset_depreciation_line
```

Comment:
Why don't you include this one in `account_asset_asset` as it's a one2many submodel?